lib/api/tenure/v1/service.ts

```diff
@@ -77,7 +77,7 @@ export interface EditTenureParams extends Partial<TenureParams> {
   id: string;
   etag: string;
   tenuredAsset?: TenureAsset | null;
-  tempAccInfo?: TemporaryAccommodationInfo;
+  tempAccInfo?: Partial<TemporaryAccommodationInfo>;
```

Comment:
Also, following a suggestion from another developer, the key name for this submodel has been changed from `tempAccInfo` to `tempAccommodationInfo` to avoid confusion with 'temporary account'.